wiz-broker/templates/wiz-broker-deployment.yaml

```diff
@@ -125,16 +122,14 @@ spec:
                 name: {{ include "wiz-broker.proxySecretName" . | trim }}
                 key: noProxyAddress
                 optional: false
+          {{- end }}
+          {{- if .Values.global.broker.skipTlsVerify }}
           - name: DISABLE_TLS_VALIDATION
-            valueFrom:
-              secretKeyRef:
-                name: {{ include "wiz-broker.proxySecretName" . | trim }}
-                key: skipTlsVerify
-                optional: false
-          {{- if .Values.global.httpProxyConfiguration.caCertificate }}
-          - name: CA_CERT_PATH
-            value: {{ $customCertificateFilePath }}
+            value: true
```

Comment:
Take the value maybe instead of setting to true?

Reply:
We only define this if `skipTlsVerify` is true, otherwise I won't define the environment variable.